curriculum/challenges/english/06-information-security-and-quality-assurance/information-security-with-helmetjs/hash-and-compare-passwords-synchronously.english.md

```diff
@@ -7,9 +7,17 @@ challengeType: 2
 ## Description
 <section id='description'>
 As a reminder, this project is being built upon the following starter project on <a href='https://glitch.com/#!/import/github/freeCodeCamp/boilerplate-bcrypt/'>Glitch</a>, or cloned from <a href='https://github.com/freeCodeCamp/boilerplate-bcrypt/'>GitHub</a>.
-Hashing synchronously is just as easy to do but can cause lag if using it server side with a high cost or with hashing done very often. Hashing with this method is as easy as calling <blockquote>var hash = bcrypt.hashSync(myPlaintextPassword, saltRounds);</blockquote>
+Hashing synchronously is just as easy to do but can cause lag if using it server side with a high cost or with hashing done very often. Hashing with this method is as easy as calling 
+```js
+var hash = bcrypt.hashSync(myPlaintextPassword, saltRounds);
+```
+
 Add this method of hashing to your code and then log the result to the console. Again, the variables used are already defined in the server so you wont need to adjust them. You may notice even though you are hashing the same password as in the async function, the result in the console is different- this is due to the salt being randomly generated each time as seen by the first 22 characters in the third string of the hash.
-Now to compare a password input with the new sync hash, you would use the compareSync method: <blockquote>var result = bcrypt.compareSync(myPlaintextPassword, hash);</blockquote> with the result being a boolean true or false.
+Now to compare a password input with the new sync hash, you would use the compareSync method: 
+```js
+var result = bcrypt.compareSync(myPlaintextPassword, hash);
+```
+ with the result being a boolean true or false.
```

Comment:
```suggestion

 with the result being a boolean true or false.
```
Just erring on the side of caution, by sticking a newline in.

Reply:
Resolved in new commit.